paymentsheet/src/main/java/com/stripe/android/paymentsheet/elements/IdealBankConfig.kt

```diff
@@ -10,7 +10,7 @@ internal class IdealBankConfig : DropdownConfig {
     override val debugLabel = "idealBank"
 
     @StringRes
-    override val label = R.string.title_bank_account
+    override val label = R.string.stripe_paymentsheet_ideal_bank
```

Comment:
Looking at AddPaymentMethodActivity seems like this string can be used for both this, basic integration, netbanking and FPX

Reply:
This string says "iDEAL Bank", that's what iOS and web use.
https://stripe-payments-demo.appspot.com/